mdformat/plugins.py

```diff
@@ -27,6 +28,11 @@ def _load_codeformatters() -> Dict[str, Callable[[str, str], str]]:
 class ParserExtensionInterface(Protocol):
     """A interface for parser extension plugins."""
 
+    def add_cli_options(self, parser: argparse.ArgumentParser) -> None:
+        """Add options to the mdformat CLI, to be stored in
+        mdit.options["mdformat"]"""
+        pass
```

Comment:
The `pass` should be redundant here, since the docstring is a Python statement. (In fact same applies to all method implementations in this class)